Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -135,30 +188,111 @@ void HiresTexture::Update()
       }
     }
 
-    s_textureCacheAbortLoading.Clear();
+    s_texturePrefetcher_AbortLoading.Clear();
     s_prefetcher = std::thread(Prefetch);
   }
 }
 
 void HiresTexture::Prefetch()
 {
   Common::SetCurrentThreadName("Prefetcher");
+  // bytes to MBytes calcs will auto convert stuff to double
+  // so why not use double for the rest and config so it's consistent with printf %.1f
+  // because this is still WIP - remove_this_when_done
+
+  double prefetch_starttime_total = 0;
+  double prefetch_stoptime_total = 0;
+
+  // Default 30% -- Todo GUI User Configurable, INI Only for now
+  double percent_mem_to_cull = 30.0;  // aka culled_percent
```

Comment:
Secondairly, I used the culled_percent in the GFX config since it's shorter, there were many variations before this was pushed, you only see what I eventually come up with, but it doesn't mean it's already set in stone.